test/utils/azure/azure_helpers.go

```diff
@@ -313,7 +313,7 @@ func getCloudConfig(env azure.Environment) cloud.Configuration {
 	}
 }
 
-func getClient(env azure.Environment, subscriptionID, tenantID string, cred *azidentity.ClientSecretCredential, scope string) *Client {
+func getClient(env azure.Environment, subscriptionID, _ string, cred *azidentity.ClientSecretCredential, scope string) *Client {
```

Comment:
if a parameter not used in func, then just remove this param in the function

Reply:
done